src/theme/themes/plh_facilitator_cw/_overrides.scss

```diff
@@ -336,6 +336,14 @@
       }
     }
 
+    ion-button[data-variant~="tab"] {
+      --background: none;
+      color: var(--ion-color-primary);
+      &:hover {
+        --background: unset;
+      }
+    }
+
```

Comment:
question(non-blocking):
It seems like most deployment are modifying the base styles. Is it a case that the base style still makes more sense for the rest of the deployments that do not include override (so aren't shown up here), or would it make more sense to update the base style to avoid the need for overrides (if these happen to be the only deployments currently using?)